### PR TITLE
Assignments template improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- list view with quickview
- ngDialog for create/update view
- candidates
- polls

Ready for first review.

### CODE REVIEW

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 1]
```diff
@@ -24,18 +24,98 @@
...
+        <translate>Unpublish result</translate>
+      </button>
+      <a ng-click="deleteBallot(poll)" class="btn btn-default btn-sm">
+        <i class="fa fa-times"></i>
+        <translate>Delete</translate>
+      </a>
+    </div>
+    <div class="results">
+      <div ng-repeat="option in poll.options">
+        <strong>User#{{ option.candidate_id }}</strong>
```
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: @ostcar How is the best way to get the full user name of candidate_id?

**Comment by emanuelschuetze** [comment]
@ostcar Please look also at #1707. Maybe you can give me a quick fix for it which I can implement into this pull request.

File: openslides/assignments/static/templates/assignments/assignment-detail.html (hunk 1 above)
Comment (ostcar) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: I think we have to create DS-Store classes for option objects. See the assignment_related_user as an example.

**Comment by ostcar** [comment]
I would like that we merge this right now so no "super-pr" like the one from motions arises.

@normanjaeckel please check an merge

**Comment by ostcar** [comment]
@normanjaeckel emanuel and I promise to fix your remarks in another pull request, as we would in this one. But we need this merged to work on it this evening.

**Comment by normanjaeckel** [comment]
Ok.No remarks at the moment.

**Pull request merged**